modules/k8s/tunnel.go

```diff
@@ -47,6 +47,14 @@ func (resourceType KubeResourceType) String() string {
 	}
 }
 
+func makeLabels(labels map[string]string) string {
```

Comment:
Can you add a comment mentioning what this is doing, or alternatively point to where it is sourced from?

Reply:
added a comment.